### PR TITLE
Add a direct reference to JSON schema

### LINKED ISSUE #61: DAP JSON schema is hard to find.
I could not find any links to machine-readable [DAP schema file](https://github.com/Microsoft/debug-adapter-protocol/blob/gh-pages/debugAdapterProtocol.json) from the [DAP website](https://microsoft.github.io/debug-adapter-protocol/).  It would be pretty hard for a developer who is not familiar with DAP's history to know that one does exist and where to find it.

### PR DESCRIPTION
**Pull request by vadimcn**
Fixes #61

### CODE REVIEW

**Comment by Takangcwele** [comment]
tee

**Review by Takangcwele — APPROVED** [review]
tee

File: specification.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ toc: true
 
 The Debug Adapter Protocol defines the protocol used between an editor or IDE and a debugger or runtime.
 
+Machine-readable JSON schema can be found [here](./debugAdapterProtocol.json).
+
 The change history of the specification can be found [here](./changelog).
```
Comment (Takangcwele) [inline specification.md:16]: t

File: specification.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ toc: true
 
 The Debug Adapter Protocol defines the protocol used between an editor or IDE and a debugger or runtime.
 
+Machine-readable JSON schema can be found [here](./debugAdapterProtocol.json).
```
Comment (Takangcwele) [inline specification.md:14]: tee

File: specification.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ toc: true
 
 The Debug Adapter Protocol defines the protocol used between an editor or IDE and a debugger or runtime.
```
Comment (Takangcwele) [inline specification.md:12]: tee

Comment (Takangcwele) [inline specification.md:12]: tee

**Comment by weinand** [comment]
@vadimcn after merging your PR I noticed (too late) that your fix lives in a machine generated file:



I've moved the change to the generator program instead.

**Comment by vadimcn** [comment]
LOL.  That line was loo long to read, obviously.

**Pull request merged**